package-lock.json

```diff
@@ -0,0 +1,16226 @@
+{
```

Comment:
You probably don't want a `package-lock.json` AND a `yarn.lock`. One of these belongs in a `.gitignore`

Reply:
You are right! I will add them to the `.gitignore` file.

Reply:
Just did.